spec/features/user_index_spec.rb

```diff
@@ -1,37 +1,29 @@
 require 'rails_helper'
+
 RSpec.feature 'User Index Page' do
-  # Create some test users and posts
-  let!(:user1) { User.create(name: 'user1', photo: 'https://instasize.com/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fmunkee%2Fimage%2Fupload%2Fw_1000%2Cc_fill%2Car_1%3A1%2Cg_auto%2Cr_max%2Fv1681855894%2Finstasize-website%2Flearn%2Fblonde-woman-selfie-instagram-influencer.webp&w=828&q=75') }
-  let!(:user2) { User.create(name: 'user2', photo: 'https://instasize.com/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fmunkee%2Fimage%2Fupload%2Fw_1000%2Cc_fill%2Car_1%3A1%2Cg_auto%2Cr_max%2Fv1681855894%2Finstasize-website%2Flearn%2Fblonde-woman-selfie-instagram-influencer.webp&w=828&q=75') }
-  let!(:post1) { Post.create(user: user1, title: 'test') }
-  let!(:post2) { Post.create(user: user2, title: 'test') }
+  let!(:user1) { User.create(name: 'Tom', bio: 'Teacher from Mexico.', photo: 'https://avatars.githubusercontent.com/u/98366229?v=4', posts_counter: 0) }
+  let!(:post1) { Post.create(author_id: user1, title: 'Hello') }
+  let!(:post2) { Post.create(author_id: user1, title: 'You are welcome') }
```

Comment:
good morning sir..
please i cant find what the required changes you commented on my pull request, please i need clarity.

Reply:
> good morning sir.. please i cant find what the required changes you commented on my pull request, please i need clarity.

Hi @maths29,
You can't see it, Because it's the project requirement and you didn't include all changes in this PR.
Anyway, I updated my comment so you can find it.
Have a great day.